### DISCUSSION TITLE
Elaborate RDF Adapter

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Informatievlaanderen/VSDS-Linked-Data-Interactions#247] [Q&A]
https://informatievlaanderen.github.io/VSDS-Linked-Data-Interactions/core/ldi-adapters/rdf-adapter

`As the most basic Adapter of the LDI Core Building Blocks, the RDF Adapter will take in an RDF string and convert it into an internal Linked Data model.`

I am wondering what does that adapter do exactly? (Maybe an example?)
And what does the internal Linked Data model mean in the current context?
Thanks

**Comment by Yalz** [comment]
Moved to issues. To be picked up
